### PR TITLE
Issue undef1

### PR DESCRIPTION
**Pull request by Anoushka444**
Removed the errors and added the ignore file.

### CODE REVIEW

**Review by jayrevolinskyjr — CHANGES_REQUESTED** [review]
My only concern is that there are a massive amount of changes to the package-lock.json, I thought it was a part of the .gitignore? Besides that and the merge issues, it looks good.

**Comment by jayrevolinskyjr** [comment]
Looks like the merge conflicts and package-lock.json problem is related to Issue [104](https://github.com/ChicoState/PantryNode/issues/104)

**Comment by AbhinavReddy-Dev** [comment]
To resolve the `package-lock.json` errors, the process is to reinstall the packages after resolving the conflicts in `package.json`. This will update the `package-lock.json` file and the conflicts are auto resolved by `npm`.

File: console.log(err))
Comment (Jooms) [inline console.log(err)):1]: Remove this file

File: git
Comment (Jooms) [inline git:1]: Remove this file

File: {
Comment (Jooms) [inline {:1]: Remove this file.

File: routes/users.js
Comment (Jooms) [inline routes/users.js:1]: nit: (meaning optional)
You didn't change anything in this file. So remove the extra line you added.

Or the cool git way
```shell
git reset main -- backend/routes/users.js
```

**Comment by Jooms** [comment]
Don't forget to rebase :)

**Review by Jooms — CHANGES_REQUESTED** [review]
Rebase so we can see the CI run

File: {
Comment (Anoushka444) [reply {:1]: Done

File: console.log(err))
Comment (Anoushka444) [reply console.log(err)):1]: Done

**Review by Anoushka444 — COMMENTED** [review]
Reviews addressed!

File: routes/users.js
Comment (Anoushka444) [reply routes/users.js:1]: Done

**Review by jayrevolinskyjr — APPROVED** [review]

**Comment by jayrevolinskyjr** [comment]
Approved, looks like the re-base worked well.

**Review by Jooms — CHANGES_REQUESTED** [review]

**Pull request merged**